standardFormats/advising_widget/readme.md

```diff
@@ -1,3 +1,9 @@
+## Important!
+
+Files only need to be provisioned for the Advising Widget if opting for **Appointments** in the recipe.
+
+They are **not required** if Appointments are not selected.
```

Comment:
The same question here about whether appointments should have the capital letter. Otherwise approved.

Reply:
All good, thanks